book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    super()
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] As requested by the previous code reviewer, I think the `add_rental` method should be something like so:
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, please kindly fix that.

Reply:
Thanks for the feedback